docs/docs/api/endpoints/messages.md

```diff
@@ -39,8 +39,16 @@ latest.
       // See glossary
       "sent_at": "{string}",
       //'yyyy-MM-dd'T'HH:mm:ss.SSSZ' date in UTC form, to be localized by clients
-      "source": "{String}"
+      "source": "{String}",
       // one of the possible sources
+      "metadata": {
+         "source": {
```

Comment:
nitpick: I'd use a different example here since this is conversation metadata: Perhaps something like `metadata.sentFrom=iphone` or something like that :)

Reply:
I actually picked this property for this reason specifically. It's metadata that the user didn't input so it doesn't seem that she/he missed a step.

I can definitely change it, it's just "funny" how we had different perceptions of the same data